modules/vaos/spec/config/initializers/flipper_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Flipper::Instrumentation::AppointmentsEventSubscriber do
+  context 'logs changes to toggle values' do
+    it 'logs warning for restricted operation of critical feature' do
+      allow(Rails.logger).to receive(:warn)
+      Flipper.disable(:va_online_scheduling_subscriber_unit_testing)
```

Comment:
Flipper.disable is no longer the recommended method for Flipper features. Please see the documentation at https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample
This is the recommended Flipper.disable ->      
`allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(false)`

Reply:
I'm not sure if the recommendation would work in this case since we are unit testing specifically for the instrumentation calls whenever `Flipper.disable/enable` are called directly rather than checking if a feature is turned on or off. 

This test uses the same pattern as those [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/lib/flipper/instrumentation/event_subscriber_spec.rb). To ensure safety when running with other tests, unit testing specific toggles that's not used anywhere else in the code `va_online_scheduling_subscriber_unit_testing` and `va_online_scheduling_this_is_only_a_test` are used. The toggling of these shouldn't have any other side effect.

Reply:
Thanks for the clarification.  Approving for this edge case.